### PR TITLE
Confirm civicrm coding standards

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

File: CRM/Fbpixel/Upgrader/Base.php [hunk 1]
```diff
@@ -8,39 +8,39 @@
 class CRM_Fbpixel_Upgrader_Base {
 
   /**
-   * @var varies, subclass of ttis
```
Comment (twomice) [inline CRM/Fbpixel/Upgrader/Base.php:11]: Hi Lucky. What was the civilint error on this line? (If you change it back and run civilint again, you should see the error message.)

Comment (lucky091588) [reply CRM/Fbpixel/Upgrader/Base.php:11]: The same as in the screenshots.
```
Expected "variessublcassofttis" but found "varies, subclass of ttis" for @var tag in member variable comment
```

Comment (twomice) [reply CRM/Fbpixel/Upgrader/Base.php:11]: Okay, thanks @lucky091588 . I think we'd better ignore this error message and leave the lines as they are, e.g.,
```
   * @var varies, subclass of ttis
```
I'm not sure why civilint is reporting this as an error; I'll try to ask the civilint developer about it. But for now, it just seems unnecessary to turn a normal English comment into a one-word string, just to pass this lint test. Thanks!

**Comment by twomice** [comment]
Merged. Thanks @lucky091588 !

**Pull request merged**